react/src/lib/components/EdsIcon/components/EdsIcon.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as edsIcons from "@equinor/eds-icons";
+import { IconData } from "@equinor/eds-icons";
+import { Icon } from "@equinor/eds-core-react";
+import { Tooltip } from "@material-ui/core";
+
+type IconProps = {
+    icon: string;
+    size?: 16 | 24 | 32 | 40 | 48;
+    color?: string;
+};
+
+export const EdsIcon: React.FC<IconProps> = (props) => {
+    let icon: IconData | undefined = undefined;
+    if (props.icon) {
+        icon = Object.values(edsIcons).find((el) => el.name === props.icon);
+    }
+
+    if (!icon) {
+        return (
+            <Tooltip
+                title={
+                    `An icon with name "${props.icon}" does not exist.` +
+                    ` Please check the icon name for typos. ` +
+                    `An overview of all available icons can be found at ` +
+                    `https://eds.equinor.com/assets/system-icons/library/.`
```

Comment:
that link is a 404, at least for me from an unmanaged laptop connected to Statoil-Approved

i managed to access the storybook for icons though, under

https://eds-storybook-react.azurewebsites.net/?path=/story/icons--preview

Reply:
Yep, seems like EDS changed their website. I will adjust the link. Thanks for the catch!

Reply:
no worries : )